proposals/lip-0074.md

```diff
@@ -920,6 +925,22 @@ def applyProposalOutcome(type: int, data: bytes) -> None:
         call updateConfig(dataObj.paramName, dataObj.value) method of module dataObj.moduleName
 ```
 
+#### validateConfigUpdate
+
+The function raises an exception if the proposed config update is invalid.
+
+```python
+def validateConfigUpdate(name: string, value: bytes):
+    type = type of value with name in configSchema
+    valueDecoded = validateAndDecode(value, type)
+    if name == "quorumPercentage":
+        if valueDecoded > 1000000:
+            raise Exception("Quorum percentage must be in parts-per-million")
+    if name == "quorumDuration":
+        if valueDecoded > configStore.voteDuration:
+            raise Exception("Quorum duration can not be greater than vote duration")
```

Comment:
Should here be some `else` indicating what happens if `name` is none of the ones stated above?

Reply:
I have added a check that name is a valid config name, so now the function should be fine.